client/src/components/Navbar/Navbar.js

```diff
@@ -1,21 +1,102 @@
-import React from 'react';
+import React from "react";
 import "./Navbar.css";
-import Logo from "../Logo/Logo"
+import Logo from "../Logo/Logo";
+import Burger from "../Burger/Burger";
+import ProfilePicture from "../ProfilePicture/ProfilePicture";
+import NavLink from "../NavLink/NavLink";
+import Icon from "../Icon/Icon";
+import PropTypes from "prop-types";
 
 // Function based React Component
-const Navbar = (props) => {
```

Comment:
This component will probably need state (like to track whether the hamburger is open or closed)

Reply:
@ZakkMan I originally had it there when I started this component but you had me remove it for some reason.